### PR TITLE
Make maker homepage a unit test instead of storybook

### PR DESCRIPTION
**Pull request by bencodeorg**
Working on [this PR](https://github.com/code-dot-org/code-dot-org/pull/49419), I saw this component did not have unit tests, but had a Storybook component. I didn't realize that we had migrated this story already to the new Storybook API.

It looks like it was sort of a [50/50 decision to keep this entry](https://github.com/code-dot-org/code-dot-org/pull/48411), but I think it's more extensible to have a unit test here than a Storybook entry (eg, can now add a unit test confirming the presence of a notification).

### CODE REVIEW

**Review by molly-moen — APPROVED** [review]

File: apps/test/unit/templates/MakerLandingTest.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  it('renders', () => {
+    const fakeTopCourse = {
+      assignableName: 'Maker',
+      lessonName: 'Maker Lesson',
+      linkToOverview: 'fakelink.com',
+      linkToLesson: 'fakelink2.com'
+    };
+
+    shallow(<MakerLanding topCourse={fakeTopCourse} />);
+  });
```
Comment (fisher-alice) [inline apps/test/unit/templates/MakerLandingTest.js:15]: Is an assertion not wanted here?

**Review by fisher-alice — APPROVED** [review]
Makes sense to use a unit test here - I was not sure about this one when I migrated the storybook. Thanks for doing this!

File: apps/test/unit/templates/MakerLandingTest.js (hunk 1 above)
Comment (bencodeorg) [reply apps/test/unit/templates/MakerLandingTest.js:15]: Correct, the "test" in these "it renders" tests is just that it's able to render (ie, shallow doesn't error).

Comment (fisher-alice) [reply apps/test/unit/templates/MakerLandingTest.js:15]: Cool!  Would it make sense to add another unit test to check the presence of a notification if included?

**Pull request merged**